### PR TITLE
Update runtime Readme

### PR DESCRIPTION
**Pull request by amityadav0**

### CODE REVIEW

File: packages/runtime/README.md [hunk 1]
```diff
@@ -1,12 +1,45 @@
 # Runtime
 
-JavaScript Algorand runtime. It provides:
+This package (JavaScript Algorand runtime) executes transactions and processes TEAL in 3 parts :-
 
-- Account state
-- TEAL stateless interpreter
-- TEAL stateful interpreter
+- [Runtime](../packages/runtime/src/runtime.ts): For a transaction or txn group, the state management is handled by the `Runtime`. User can use `Runtime` object to set up accounts, create applications, opt-in to app, update app, etc...
+- [StoreAccount](../packages/runtime/src/account.ts): User can create new accounts using the `StoreAccount` object. All information about account (`apps`, `assets`, `localState`, `globalState` etc..) is stored in `StoreAccount`.
```
Comment (robert-zaremba) [inline packages/runtime/README.md:6]:
```suggestion
- [StoreAccount](../packages/runtime/src/account.ts): user can create new accounts using a `StoreAccount` object. All information about an account (`apps`, `assets`, `localState`, `globalState` etc..) is stored in `StoreAccount`.
```

File: packages/runtime/README.md [hunk 2]
```diff
@@ -1,12 +1,45 @@
...
+- [Parser](../packages/runtime/src/parser): Reads TEAL code and converts it to a list of opcodes which are executable by the interpreter. If any opcode/data in teal code is invalid, parser will throw an error.
```
Comment (robert-zaremba) [inline packages/runtime/README.md:7]:
```suggestion
- [Parser](../packages/runtime/src/parser): reads TEAL code and converts it to a list of opcodes which are executable by the interpreter. If a teal code contains an invalid opcode/data, parser will throw an error.
```

File: packages/runtime/README.md [hunk 3]
```diff
@@ -1,12 +1,45 @@
...
+- [Interpreter](../packages/runtime/src/interpreter): Executes the list of opcodes returned by parser and updates stack after each execution. At the end of execution, if the stack contains a single non-zero uint64 element then the teal code is approved, and transaction can be executed.
```
Comment (robert-zaremba) [inline packages/runtime/README.md:8]:
```suggestion
- [Interpreter](../packages/runtime/src/interpreter): executes a list of opcodes returned by parser. Interpreter creates teal execution stack and interacts with runtime to perform changes related a TEAL code execution. At the end of execution, if the stack contains a single non-zero uint64 element then the teal code is approved, and transaction can be executed.
```

File: docs/testing-teal.md [hunk 4]
```diff
@@ -256,14 +260,11 @@ Currently, `runtime` supports:
...
- - Asset related transactions:
-    - Asset Destroy
-    - Asset Revoke
-    - Asset Freeze
-    - Asset Configuration
+- Asset related transactions:
+  - Asset Destroy
+  - Asset Revoke
+  - Asset Freeze
+  - Asset Configuration
```
[the remark this answers is not in the document]
Comment (amityadav0) [reply docs/testing-teal.md:267]: yes

File: packages/algob/README.md [hunk 5]
```diff
@@ -1,13 +1,12 @@
...
 
 This package provides the main entry point into the application.
 
 Please read the main [README](https://github.com/scale-it/algo-builder/blob/master/README.md) file for details about the project and `algob`. This file provides a development description about the `algob` package.
 
 ## Usage
 
-
-`algob` can be included as a library by importing `internal/lib/lib.ts` or can be run from command line as described in the project [README](https://github.com/scale-it/algo-builder/blob/master/README.md) file.
+`algob` can be included as a library using `yarn add @algo-builder/algob` and then import it using `import * from '@algo-builder/algob'` or can be run from command line as described in the project [README](https://github.com/scale-it/algo-builder/blob/master/README.md) file.
```
[the remark this answers is not in the document]
Comment (amityadav0) [reply packages/algob/README.md:9]: ok

**Review by man-of-code — APPROVED** [review]
LGTM

File: packages/runtime/README.md [hunk 6]
```diff
@@ -1,12 +1,46 @@
 # Runtime
 
-JavaScript Algorand runtime. It provides:
+This package implements a light version of Algorand runtime. It allows to executes transactions and processes TEAL in JavaScript environment. The packages provides 4 main object:
 
-- Account state
-- TEAL stateless interpreter
-- TEAL stateful interpreter
+- [Runtime](../packages/runtime/src/runtime.ts): handles transaction or txn group processing and state management . User can use a `Runtime` object to set up accounts, create applications, opt-in to app, update app, etc...
```
Comment (ratik21) [inline packages/runtime/README.md:5]: Please update the link here (`../packages/runtime/src/*` -> `./src/*`)

File: packages/runtime/README.md [hunk 7]
```diff
@@ -1,12 +1,46 @@
...
+    - [Asset Revoke](https://developer.algorand.org/docs/reference/transactions/#asset-clawback-transaction)
+    - [Asset Freeze](https://developer.algorand.org/docs/reference/transactions/#asset-freeze-transaction)
+    - [Asset Configuration](https://developer.algorand.org/docs/reference/transactions/#asset-configuration-transaction)
 
 STATUS: production ready.
 
-## Usage
+## Contributing
+
+Please read the main [README](https://github.com/scale-it/algo-builder/blob/master/README.md).
```
Comment (ratik21) [inline packages/runtime/README.md:46]: Can we also add a coming soon section? Eg.
```
## Coming Soon

TEAL v3 Support
```

File: packages/runtime/README.md (hunk 6 above)
Comment (amityadav0) [reply packages/runtime/README.md:5]: Ok

File: packages/runtime/README.md (hunk 7 above)
Comment (amityadav0) [reply packages/runtime/README.md:46]: yes, good idea

**Pull request merged**